Update readme (#2)
Tetrax-10/tails-tweaks: README.md

![](https://raw.githubusercontent.com/Tetrax-10/tails-tweaks/main/assets/showcase.jpg)

## Features

> - **Dark Mode**: Switches the desktop and terminal to dark mode.

Reviewer: ```suggestion
- **Dark Mode**: Sets the system theme to dark mode.
```

- **Wallpaper**: Changes the desktop wallpaper to a specified image.
- **Auto Sleep**: Disables automatic sleep mode.

> - **Screen Dim & Lock**: Prevents screen dimming and automatic locking.

Reviewer: separate them, like I said in the other PR.

Everytime you boot into Tails, do step 3.

> #### Paranoid Installation
> 
> This method is more secure because it does not require you to clone the repository or download the image files. 
> However, you will need to manually create the `wallpapers` folder and place an image file of your choice in it.
> 
> 1. Open configure.sh in this GitHub repository in the browser and copy the entire contents.
> 2. Paste the contents into a new file called `configure.sh` in the `Persistent` folder.
> 3. In the Terminal, run `chmod +x configure.sh` to make the script executable.
> 4. Modify the script to your preferences according to the usage instructions below.
> 5. Run `configure.sh` to apply the changes.

Reviewer: ```suggestion
**Note**: Downloading files from the internet is not a wise option when privacy and security are your top priorities. If you really want to be secure, do not trust files from the internet; they may contain stegomalware (hidden malicious code inside a file). I recommend not trusting files from random internet users (including me). For installation I recommend you to manually create the `configure.sh` file by copying its contents from GitHub, and download your wallpaper from trusted sites. Make sure to use [VirusTotal](https://www.virustotal.com/gui/) to check that it is not infected.
```

Reviewer: You can add "Modify the script to your preferences according to the usage instructions below." step to #Installation

## Usage

1. **Set Variables**: Edit the script to set your preferences by modifying the variables at the top. Each setting can be enabled or disabled by setting the corresponding variable to `true` or `false`.

   ```bash
   dark_mode=true

>    dark_terminal=true

Reviewer: merge `dark_terminal` with `dark_mode`.

   change_wallpaper=true
   wallpaper_path=/home/amnesia/Persistent/wallpapers/wallpaper.jpg
   auto_sleep=true

>    dim_screen=true

Reviewer: add `auto_screen_lock`

   disable_camera=true
   disable_microphone=true
   disable_location=true

>    disable_usb_auto_mount=false

Reviewer: set `disable_usb_auto_mount` to true

    <td>After</td>
  </tr>
  <tr align="center">
    <td>
      <img alt="Before" src="https://raw.githubusercontent.com/Tetrax-10/tails-tweaks/main/assets/before.jpg" style="width: 400px;">
    </td>
    <td>
      <img alt="After" src="https://raw.githubusercontent.com/Tetrax-10/tails-tweaks/main/assets/after.jpg" style="width: 400px;">
    </td>
  </tr>
</table>

> ## License
> 
> This script is open-source and available under the [MIT License](LICENSE). Feel free to modify and share it.
> 

Reviewer: ```suggestion
```
